tests/lib/rules/destructuring-assignment.js

```diff
@@ -193,14 +205,6 @@ ruleTester.run('destructuring-assignment', rule, {
     errors: [
       {message: 'Must never use destructuring props assignment in SFC argument'}
     ]
-  }, {
-    code: `const MyComponent = (props, { color }) => (
-      <div id={props.id} className={props.className} />
-    );`,
-    options: ['never'],
-    errors: [
-      {message: 'Must never use destructuring context assignment in SFC argument'}
-    ]
```

Comment:
why is this test changed? this implies it's a breaking change.

Reply:
i think that case is legacy context assignment which is not used in current react version.
sorry for change without any comments.

Reply:
This plugin supports all react versions; nothing can be removed.

Reply:
i'm sorry. actually i want to ask about that. my mistake. then should I edit and make new PR?

plus,  ``const destructuringContext = node.params && node.params[1] && node.params[1].type === 'ObjectPattern';``should be added in ``function handleStatelessComponent(node)``  right?

Reply:
@ljharb 
i did revive test cases which i deleted and rules associated. please check again